ports/brotato/README.md

```diff
@@ -0,0 +1,14 @@
+## Notes
+
+Thanks to [Blobfish Games](https://store.steampowered.com/app/1942280/Brotato/?curator_clanid=44280697) for making this wonderful roguelite
+
+The DLC isnt supported because it requires to patch out steam and that would make it easy to pirate the dlc
```

Comment:
Let's keep it simple, just say it's incompatible, thus avoiding any potential question on how to patch out steam...

```suggestion
The DLC from Steam are not compatible with the Epic version of the game that is used in this port.
```